### PR TITLE
Ignore if global_operation has no class

### PR DESCRIPTION
**Pull request by aschempp**

### CODE REVIEW

**Review by Toflar — DISMISSED** [review]

File: core-bundle/src/Resources/contao/classes/DataContainer.php [hunk 1]
```diff
@@ -1110,7 +1110,7 @@ protected function generateGlobalButtons()
 				$href = $this->addToUrl($v['href'] ?? '');
 			}
 
-			$return .= '<a href="' . $href . '" class="' . $v['class'] . '" title="' . StringUtil::specialchars($title) . '"' . $attributes . '>' . $label . '</a> ';
+			$return .= '<a href="' . $href . '" class="' . ($v['class'] ?? '') . '" title="' . StringUtil::specialchars($title) . '"' . $attributes . '>' . $label . '</a> ';
```
Comment (leofeyer) [inline core-bundle/src/Resources/contao/classes/DataContainer.php]: Shouldn‘t we omit the empty `class=""` attribute in this case as well?

Comment (aschempp) [reply core-bundle/src/Resources/contao/classes/DataContainer.php]: we could, but it would be more overhead so I thought we'd just keep it simple here 😅

Comment (leofeyer) [reply core-bundle/src/Resources/contao/classes/DataContainer.php]:
```suggestion
			$return .= '<a href="' . $href . '"' . (isset($v['class']) ? ' class="' . $v['class'] . '"' : '') . ' title="' . StringUtil::specialchars($title) . '"' . $attributes . '>' . $label . '</a> ';
```

Doesn‘t seem like too much overhead to me. 😉

**Review by leofeyer — APPROVED** [review]

**Comment by leofeyer** [comment]
Thank you @aschempp.

**Pull request merged**